RestaurantReservation.Db/Services/CustomerService.cs

```diff
@@ -1,4 +1,4 @@
-﻿using RestaurantReservation.Db.Models;
+﻿using RestaurantReservation.Db.Entities;
 using RestaurantReservation.Db.Repositories;
 
 namespace RestaurantReservation.Db.Services
```

Comment:
All these services shouldn't be in the DB layer, we should introduce a new layer `RestaurantReservation.Application` that contains all these services and business logic, and this layer shouldn't have directly access to the db context, it will just be dealing with the repos through the interfaces, this way you are decoupling the business logic from the db layer.